src/AI.java

```diff
@@ -221,27 +221,23 @@ static int bestColumn(Board board, int difficulty) {
 			}
 			return temp;
 		} else if (difficulty == 1) { // modify depth based on current difficulty
-			depth = 2;
+			depth = 1;
 		} else if (difficulty == 2) {
-			depth = 3;
+			depth = 2;
 		} else {
-			depth = 4;
+			depth = 3;
 		}
-		ArrayList<Integer> bestRows = minMax(board, depth, 1, Integer.MIN_VALUE, Integer.MAX_VALUE); // grab value from big algorithm
-		System.out.println(Arrays.toString(bestRows.toArray()));
+		TreeMap<Integer, Integer> bestRows = minMax(board, depth, 1, Integer.MIN_VALUE, Integer.MAX_VALUE); // grab value from big algorithm
```

Comment:
What's the reason for using a TreeMap versus a HashMap?

Reply:
Easier to iterate through, but a hashmap works too and it should be faster now that I think about it.

Reply:
HashMap is O(1) for inserting, grabbing, and removing on average, while TreeMap is O(log(n))